babel.config.js

```diff
@@ -1,3 +1,13 @@
 module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-react'],
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+    '@babel/preset-react',
+  ],
```

Comment:
We should be able to simplify this by just using CRA's Babel config directly. It would also make our build tooling more consistent with how CRA would be used after generating the template.

```suggestion
    presets: ['react-app'],
```

Reply:
Good point